### DISCUSSION TITLE
The tpv33 calculation result is invalid and the tpv5 calculation program terminates

### DISCUSSION THREAD

**Discussion by dxycugb** [SeisSol/SeisSol#1062] [Q&A]
Hi, I am trying to run the benchmark of seissol on my personal computer but I have encountered problems.

1. About tpv33:

I changed Format = 6 in the parameter file to enable hdf5 output, but the tpv33 output file I got when paraview visualized didn't seem to have anything available.
![截图 2024-02-25 02-33-57](https://github.com/SeisSol/SeisSol/assets/145737011/0fcb1997-3778-4dc4-82bf-bde6b030f23e)
Also, how should I use viewrec to visualize dat files?

The parameter file of tpv33 is officially provided and only Format = 6 has been modified to enable 3D wave field output.I'm not sure if the log I provided is what you need. Parameter files, log files, and output xdmf files are attached.

2.About tpv5:

I managed to run tpv33 with order 3 but the results did not seem correct, however the program was aborted when I tried to run tpv5 with order 2 and 3. I'm not sure if it's a lack of memory issue, and I haven't tried running it on a computer with more memory.
The parameter file is also given by the official, only modify Format = 6.
Output log:
dxy@pc:~/Examples/tpv5$ ./SeisSol_Release_dhsw_2_elastic parameters.par
Wed Feb 21 22:13:07, Info:  Welcome to SeisSol 
Wed Feb 21 22:13:07, Info:  Copyright (c) 2012 - 2024  SeisSol Group 
Wed Feb 21 22:13:07, Info:  Version: v1.1.2-163-gf21db452 
Wed Feb 21 22:13:07, Info:  Built on: Feb 21 2024 20:23:04 
Wed Feb 21 22:13:07, Info:  Last commit: f21db4527089c69668d8328ce6d3656ef74c2742 at 2024-01-26T14:13:16+00:00 
Wed Feb 21 22:13:07, Info:  Running on: "pc" 
Wed Feb 21 22:13:07, Info:  Using MPI with #ranks: 1 
Wed Feb 21 22:13:07, Info:  Using asynchronous MPI routines. 
Wed Feb 21 22:13:07, Info:  Using OMP with #threads/rank: 16 
Wed Feb 21 22:13:07, Info:  OpenMP worker affinity (this process): "0123456789|012345" 
Wed Feb 21 22:13:07, Info:  OpenMP worker affinity (this node)   : "0123456789|012345" 
Wed Feb 21 22:13:07, Info:  Using polling for advancing MPI communication, due to having only one MPI rank running. 
Wed Feb 21 22:13:07, Info:  The stack size ulimit is 8192 [kiB] ( = 8.0000 MiB ). 
Wed Feb 21 22:13:07, Warn:  Stack size of 8192 [kiB] ( = 8.0000 MiB ) is lower than recommended minimum of 2097152 [kiB] ( = 2.0000 GiB ). You can increase the stack size by running the command: ulimit -Ss unlimited. 
Wed Feb 21 22:13:07, Info:  Reading SeisSol parameter file... 
Wed Feb 21 22:13:07, Info:  In your parameter file, you have specified a non-positive interval for "checkpointinterval" . This mechanism is deprecated and may be removed in a future version of SeisSol. Consider disabling the whole module by setting "checkpoint" to 0 instead by adding "checkpoint = 0" to the "output" section of your parameter file instead. 
Wed Feb 21 22:13:07, Info:  The field "faultoutputflag" is no longer in use. You may safely remove it from your parameters file. 
Wed Feb 21 22:13:07, Info:  SeisSol parameter file read successfully. 
Wed Feb 21 22:13:07, Info:  Model information: 
Wed Feb 21 22:13:07, Info:  Elastic model: yes 
Wed Feb 21 22:13:07, Info:  Viscoelastic model: no 
Wed Feb 21 22:13:07, Info:  Anelastic model: no 
Wed Feb 21 22:13:07, Info:  Poroelastic model: no 
Wed Feb 21 22:13:07, Info:  Anisotropic model: no 
Wed Feb 21 22:13:07, Info:  Plasticity: no 
Wed Feb 21 22:13:07, Info:  Begin init mesh. 
Wed Feb 21 22:13:07, Info:  Mesh file: "tpv5_f200m.puml.h5" 
Wed Feb 21 22:13:07, Info:  Skipping mini SeisSol (SeisSol is used with a single rank only). 
Wed Feb 21 22:13:07, Info:  Reading PUML mesh 
Wed Feb 21 22:13:07, Info:  Found 1724102 cells 
Wed Feb 21 22:13:07, Info:  Found 283077 vertices 
Wed Feb 21 22:13:15, Info:  Computing LTS weights. 
Wed Feb 21 22:13:18, Info:  Limiting number of clusters to 2147483646 
Wed Feb 21 22:13:18, Info:  Computing LTS weights. Done.  (6840 reductions.)
Wed Feb 21 22:13:18, Info:  Using the Parmetis partition library and strategy. 
Wed Feb 21 22:13:30, Info:  PUML mesh read in: 23.8525 s (min: 23.8525 s, max: 23.8525 s)
Wed Feb 21 22:13:30, Info:  The mesh has been read. Starting post processing. 
Wed Feb 21 22:13:30, Info:  Extracting fault information. 
Wed Feb 21 22:13:31, Info:  Exchanging ghostlayer metadata. 
Wed Feb 21 22:13:31, Info:  Mesh initialized in: 24.4142 s (min: 24.4142 s, max: 24.4142 s)
Wed Feb 21 22:13:31, Info:  End init mesh. 
Wed Feb 21 22:13:31, Info:  Begin init model. 
Wed Feb 21 22:13:31, Info:  Initialize LTS. 
Wed Feb 21 22:13:31, Info:  Deriving clusters ids for min. time step width / multiRate: 0.000968002 / 2 
Wed Feb 21 22:13:31, Info:  Due to wiggle factor of 1 the minimum timestep size is reduced to 0.000968002 
Wed Feb 21 22:13:31, Info:  Number of elements in time clusters: 
Wed Feb 21 22:13:31, Info:  0: 27379 
Wed Feb 21 22:13:31, Info:  1: 585166 
Wed Feb 21 22:13:31, Info:  2: 561365 
Wed Feb 21 22:13:31, Info:  3: 299070 
Wed Feb 21 22:13:31, Info:  4: 175181 
Wed Feb 21 22:13:31, Info:  5: 75064 
Wed Feb 21 22:13:31, Info:  6: 877 
Wed Feb 21 22:13:31, Info:  maximum theoretical speedup (compared to GTS): 4.89026 per cell LTS, 3.37401 with the used clustering. 
Wed Feb 21 22:13:31, Info:  Number of elements in dynamic rupture time clusters: 
Wed Feb 21 22:13:31, Info:  0 (dr): 773 
Wed Feb 21 22:13:31, Info:  1 (dr): 25707 
Wed Feb 21 22:13:31, Info:  2 (dr): 0 
Wed Feb 21 22:13:31, Info:  3 (dr): 0 
Wed Feb 21 22:13:31, Info:  4 (dr): 0 
Wed Feb 21 22:13:31, Info:  5 (dr): 0 
Wed Feb 21 22:13:31, Info:  6 (dr): 0 
Wed Feb 21 22:13:31, Info:  Initialize Friction Model 
已杀死(Killed)

attachments:
[tpv33+tpv5.zip](https://github.com/SeisSol/SeisSol/files/14394159/tpv33%2Btpv5.zip)

**Comment by sebwolf-de** [comment]
1. It is to be expected to see blue box in Paraview initially. Choose another quantity and another time step. I can't visualize the xdmf files without the *.bin files.
2. I am not surprised that a mesh with 1.7M elements does not work on a workstation. Try to go for <200k elements.

Since this is not a problem of SeisSol, but a question about the proper usage of SeisSol, I am goint to convert this issue to a discussion.
